store/sqlstore/store.go

```diff
@@ -1070,6 +1081,20 @@ func (ss *SqlStore) RecycleDBConnections(d time.Duration) {
 }
 
 func (ss *SqlStore) Close() {
+	mlog.Debug("Waiting for active async migrations to finish...")
+	ss.wg.Wait()
```

Comment:
I think `systemd` will kill a process if it doesn't shut down in a timely fashion. Are we assuming the server stays up long enough to finish the upgrade? Might this be problematic in a cloud environment with pods that are expected to get killed more frequently?

Reply:
yeap it will kill the parent process, this implementation does not change those semantics because we still have in master a situation that if you add an index on the posts table, for example, the boot up will hand until a timeout is reached. The strategy that we've followed with some customers/cloud is telling them to run a script to add the index before they boot the server

Reply:
Wondering if we shouldn't forcibly stop our attempted migration here instead of holding on "just a bit longer" in the hopes we don't get killed anyway.